packages/admin/src/pages/super-admin-dashboard/user/[id]/block-user.page.tsx

```diff
@@ -17,13 +18,19 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     await updateUserRoles(userId, [], jwt);
   }
 
-  async function fetchPageData(jwt: string) {
-    const user = await getUserById(userId, jwt);
+  const fetchPageData = async (jwt: string) => {
+    const jwtUser = await getUserFromJwt(jwt);
```

Comment:
what's the advantage of going out to user service to do this? we already have the jwt, can we not just do it here?